### PR TITLE
Fix for zf3

### PR DESCRIPTION
**Pull request by realmanuelgeil**

### CODE REVIEW

**Comment by exptom** [comment]
@juriansluiman If I tidied up this PR to fix tests would you be happy to tag a new version of this module for use with zf3?

File: composer.json [hunk 1]
```diff
@@ -26,14 +26,10 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "zendframework/zend-mvc": "~2.2",
-        "zendframework/zend-servicemanager": "~2.2",
-        "zendframework/zend-stdlib": "~2.2",
-        "slm/queue": "0.4.*",
+        "slm/queue": "*",
```
Comment (Danielss89) [inline composer.json:29]: This is not good. Please pin it to current release.

**Review by Danielss89 — COMMENTED** [review]
I will be testing this PR the coming week.

**Comment by Danielss89** [comment]
@freddydiekante Can you fix tests?
I will test out this PR in the coming week, and if it works, we can merge and tag new release.

**Comment by imonteiro** [comment]
@Danielss89 Will be possible to merge this PR? Thanks.

**Pull request closed without merging**